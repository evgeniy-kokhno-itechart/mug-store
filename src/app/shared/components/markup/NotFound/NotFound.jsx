import React from 'react';
import './NotFound.scss';

const NotFound = () => <h1 className="not-found">Not Found</h1>;

export default NotFound;
