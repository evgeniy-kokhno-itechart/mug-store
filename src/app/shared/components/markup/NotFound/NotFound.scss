.not-found {
    text-align: center;
    font-size: 3rem;
    margin-top: 2rem;
}