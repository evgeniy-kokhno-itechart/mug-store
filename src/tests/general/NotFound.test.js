/* eslint-disable no-undef */
import React from 'react';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import NotFound from '../../general/components/NotFound';

configure({ adapter: new Adapter() });

describe('<NotFound />', () => {
  let NotFoundWrapper;

  beforeEach(() => {
    NotFoundWrapper = shallow(<NotFound />);
  });

  it('check NotFound renders properly', () => {
    expect(NotFoundWrapper).toMatchSnapshot();
  });
});
