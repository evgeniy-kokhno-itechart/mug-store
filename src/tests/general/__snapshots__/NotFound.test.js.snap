// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> check NotFound renders properly 1`] = `
<h1
  className="text-center mt-3"
>
  Not Found
</h1>
`;
